internal/vendor/wiz.go

```diff
@@ -2117,6 +2148,13 @@ type CloudAccountConnection struct {
 	TotalCount int                 `json:"totalCount"`
 }
 
+// KubernetesClusters struct
+type KubernetesClusters struct {
```

Comment:
This is represented in the GraphQL schema as KubernetesClusterConnection.  Rename to match GraphQL schema.

```suggestion
// KubernetesClusterConnection struct
type KubernetesClusterConnection struct {
```